src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
+  const { slug } = useParams();
+  let content;
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(resPeople => setPeople(resPeople))
+      .catch(() => setLoadingError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
+  if (loading) {
+    content = <Loader />;
+  } else if (loadingError) {
```

Comment:
don't use else if here

only if

Reply:
refactor please this code

Reply:
I'll try to use if, but many tests skip and 1 failed